circleguard/gui.py

```diff
@@ -973,14 +1248,14 @@ def reload_theme(self):
         switch_theme(get_setting("dark_theme"))
 
     def _open_settings(self):
-        overwrite_config()  # generate file with latest changes
+        overwrite_config() # generate file with latest changes
```

Comment:
I suggest that we should follow PEP 8 (two spaces before inline comment)

Reply:
I am frankly a bit shocked to find that pep 8 recommends two spaces; I've never followed that myself and can't recall noticing other code that does. The reasoning seems to be to make the comments stand out, but with editors displaying comments in a softer font already, that seems like a nonissue. I would prefer to not have to type twice the spaces when making a comment

Reply:
Sure, but please change all comments.

Reply:
forgot there were more; changed them now